rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/profile/TbDeviceProfileNode.java

```diff
@@ -54,8 +54,9 @@
         relationTypes = {"Alarm Created", "Alarm Updated", "Alarm Severity Updated", "Alarm Cleared", "Success", "Failure"},
         configClazz = TbDeviceProfileNodeConfiguration.class,
         nodeDescription = "Process device messages based on device profile settings",
-        nodeDetails = "Create and clear alarms based on alarm rules defined in device profile. The output relation type is either " +
-                "'Alarm Created', 'Alarm Updated', 'Alarm Severity Updated' and 'Alarm Cleared' or simply 'Success' if no alarms were affected.",
+        nodeDetails = "Create and clear alarms based on alarm rules defined in device profile.<br><br>" +
+                "Output connections: <code>Alarm Created</code>, <code>Alarm Updated</code>, <code>Alarm Severity Updated</code>, " +
+                "<code>Alarm Cleared</code>, <code>Failure</code> or <code>Success</code> - if no alarms were affected.",
```

Comment:
Failure - if an unexpected error occurs during message processing.

Reply:
I guess that we need to add something like this:

Create and clear alarms based on alarm rules defined in the device profile.
Output connections:
Alarm Created, Alarm Updated, Alarm Severity Updated -  when an alarm is created, updated, or its severity is changed based on the alarm creation conditions defined in the alarm rules.
Alarm Cleared - when the alarm is cleared based on the alarm clear conditions defined in the alarm rules.
Failure - if an unexpected error occurs during message processing.
Success - if no alarms were affected.